core/templates/pages/exploration-player-page/layout-directives/learner-local-nav.component.html

```diff
@@ -50,10 +50,62 @@
     </div>
   </li>
 </ul>
+<div ngbDropdown class="dropdown d-lg-none d-flex justify-content-end" [ngClass]="{'dropdown-border-active': isDropdownOpen}" (openChange)="toggleDropdown()">
+  <button type="button" class="mobile-option-btn btn btn-sm border border-white text-white" id="dropdownBasic1" ngbDropdownToggle>
+    <i class="fas fa-ellipsis-v exploration-player-navbar-icons"></i>
+  </button>
+  <div ngbDropdownMenu aria-labelledby="dropdownBasic1" class="dropdown-menu-right">
+    <span ngbDropdownItem tabindex="0" placement="bottom" (click)="toggleAttributionModal()" (keydown.enter)="toggleAttributionModal()" class="custom-dropdown-item">
+      <i class="fas fa-share-alt exploration-player-navbar-icons"></i>
+      <b class="icon-text">{{ 'I18N_PLAYER_SHARE_TOOLTIP' | translate }}</b>
+      <span class="oppia-icon-accessibility-label">{{ 'I18N_PLAYER_SHARE_TOOLTIP' | translate }}</span>
+    </span>
+    <span ngbDropdownItem *ngIf="feedbackOptionIsShown" tabindex="0" ngbPopover="popContent" triggers="manual" (click)="togglePopover()" (keydown.enter)="togglePopover()" #feedbackPopOver="ngbPopover" [autoClose]="false" class="custom-dropdown-item">
+      <i class="fas fa-comment-alt exploration-player-navbar-icons"></i>
+      <b class="icon-text">{{ 'I18N_PLAYER_FEEDBACK_TOOLTIP' | translate }}</b>
+      <span class="oppia-icon-accessibility-label">{{ 'I18N_PLAYER_FEEDBACK_TOOLTIP' | translate }}</span>
+    </span>
+    <a ngbDropdownItem *ngIf="canEdit" href="/create/{{explorationId}}" tabindex="0" target="_blank" rel="noopener" class="custom-dropdown-item">
+      <i class="fas fa-pen exploration-player-navbar-icons"></i>
+      <b class="icon-text">{{ 'I18N_PLAYER_EDIT_TOOLTIP' | translate }}</b>
+      <span class="oppia-icon-accessibility-label">{{ 'I18N_PLAYER_EDIT_TOOLTIP' | translate }}</span>
+    </a>
+    <span ngbDropdownItem *ngIf="username" tabindex="0" (click)="showFlagExplorationModal()" (keydown.enter)="showFlagExplorationModal()" class="custom-dropdown-item">
+      <i class="fas fa-flag exploration-player-navbar-icons"></i>
+      <b class="icon-text">{{ 'I18N_PLAYER_REPORT_TOOLTIP' | translate }}</b>
+      <span class="oppia-icon-accessibility-label">{{ 'I18N_PLAYER_REPORT_TOOLTIP' | translate }}</span>
+    </span>
+  </div>
+</div>
 
 <style>
   .exploration-player-navbar-icons {
     font-size: 21px;
     margin: 3px;
   }
+
+  .custom-dropdown-item {
+    align-items: center;
+    display: flex;
+  }
+
+  .icon-text {
+    font-size: 14px;
+    margin-left: 10px;
+  }
+
+  .mobile-option-btn {
+    border: none !important;
+    margin: .7em 0;
+    outline: none !important;
+  }
+
+  .mobile-option-btn:focus {
+  box-shadow: none;
+  }
+
+  .dropdown-border-active .mobile-option-btn {
+    border: solid #fff !important;
+    outline: #fff !important;
```

Comment:
Please avoid using !important in the code. Feel free to use more specific classes if required.

Reply:
Hi @kevintab95 
Using `!important` in CSS is because the kebab icon is from ngbootstrap which has default feedback and we need to iver-write the CSS manually.

Thanks.

Reply:
@hrshkshri You should be able to use a more specific CSS selector for this, rather than overwriting with `!important`. (You might want to read up on CSS specificity.)

Reply:
@hrshkshri Could you please point to the line in ngbootstrap that does this so that @kevintab95 can take a look? In general if you are making assertions try to provide evidence since it helps for investigation.

Reply:
Hi @seanlip @kevintab95 
sorry for the late reply, (I was working on a migration issue)

Here is a reason for using `!important` (which is indeed bad practice).
 
 removing `!important` 
![Screenshot 2024-02-19 at 10 34 58 AM](https://github.com/oppia/oppia/assets/108923011/8322b57d-45f9-4bfc-9560-1acfb84c72d4)

leads to 

![Screenshot 2024-02-19 at 10 39 51 AM](https://github.com/oppia/oppia/assets/108923011/6cd3f004-6af2-4099-896b-d0a49d8fc5e9)


because of this particular class `border` from `third party css`

![Screenshot 2024-02-19 at 10 40 29 AM](https://github.com/oppia/oppia/assets/108923011/5d359e35-0c79-47db-8ef7-890ba9f3d4c1)

Also, 
whenever we click on `button` we need to show the border to let the user know of the active-inactive button difference.

![Screenshot 2024-02-19 at 10 46 32 AM](https://github.com/oppia/oppia/assets/108923011/19ff85b7-f699-4526-a1ff-f034cee1d3a7)

`btn: focus` shows a black colour on every click.

Although, the CSS class is from `oppia.css` not from any `third party css` therefore, the `!important` tag would not be necessary for the manipulation of this behaviour.

will be using a different approach to remove the `!important` tag.